var/spack/packages/apex/package.py

```diff
@@ -0,0 +1,30 @@
+from spack import *
+from spack.util.environment import *
+
+class Apex(Package):
+    homepage = "http://github.com/khuck/xpress-apex"
+    #url      = "http://github.com/khuck/xpress-apex/archive/v0.1-release-candidate.tar.gz"
+    url      = "http://github.com/khuck/xpress-apex"
+
+    #version('0.1', '6e039c224387348296739f6bf360d081')
+    version('master', branch='master', git='https://github.com/khuck/xpress-apex.git')
```

Comment:
Sure thing. By the way, I was just following the example in rose/package.py - in case that one needs a commit, too.